tools/serverpod_cli/lib/src/database/create_definition.dart

```diff
@@ -142,16 +142,17 @@ String? _getColumnDefault(
       DateTime? dateTime = DateTime.parse(defaultValue);
       return '\'${DateFormat('yyyy-MM-dd HH:mm:ss').format(dateTime)}\'::timestamp without time zone';
     case DefaultValueAllowedType.bool:
-      var defaultValue = column.defaultPersistValue;
       return defaultValue;
     case DefaultValueAllowedType.int:
-      var defaultValue = column.defaultPersistValue;
       return '$defaultValue';
     case DefaultValueAllowedType.double:
-      var defaultValue = column.defaultPersistValue;
       return '$defaultValue';
     case DefaultValueAllowedType.string:
-      var defaultValue = column.defaultPersistValue;
       return '$defaultValue::text';
+    case DefaultValueAllowedType.uuidValue:
+      if (defaultUuidValueRandom == defaultValue) {
+        return 'gen_random_uuid()';
+      }
+      return '$defaultValue::uuid';
```

Comment:
Question: Changes here look a bit funky, why could all `var defaultValue = column.defaultPersistValue;` be removed?

Reply:
Because we already have `var defaultValue = column.defaultPersistValue;` on line 129
![Screenshot 2024-08-15 at 9 11 07 AM](https://github.com/user-attachments/assets/7df810a2-90a2-4d4c-a457-093e41abd842)